### PR TITLE
Fix div by zero error from prior rescaling

### PR DESCRIPTION
**Pull request by lukesonnet**
### Features and Changes

- Closes **(add link to issue here)**

### CODE REVIEW

**Review by bryce-fitzsimons — APPROVED** [review]

File: packages/stats/gbstats/bayesian/tests.py [hunk 1]
```diff
@@ -158,6 +144,23 @@ def compute_result(self):
...
+        # rescale prior if needed
+        scaled_prior_effect = self.config.prior_effect
+        if self.relative and self.config == "absolute":
+            scaled_prior_effect = GaussianPrior(
+                self.config.prior_effect.mean / abs(self.stat_a.unadjusted_mean),
+                self.config.prior_effect.variance / pow(self.stat_a.unadjusted_mean, 2),
+                self.config.prior_effect.proper,
+            )
+        elif not self.relative and self.config.prior_type == "relative":
+            if self.config.prior_effect.proper and self.stat_a.unadjusted_mean <= 0:
```
Comment (lukebrawleysmith) [inline packages/stats/gbstats/bayesian/tests.py:156]: couldn't we scale by absolute value here if unadjusted_mean < 0?  We take absolute value when scaling below for the positive case.  So we should either permit unajudsted_mean < 0, or remove the abs() call below.

File: packages/stats/gbstats/bayesian/tests.py [hunk 2]
```diff
@@ -174,17 +177,16 @@ def compute_result(self):
             self.stat_a.unadjusted_mean,
         )
 
-        post_prec = (
-            1 / data_variance
-            + int(self.prior_effect.proper) / self.prior_effect.variance
+        post_prec = 1 / data_variance + (
+            0 if not scaled_prior_effect.proper else 1 / scaled_prior_effect.variance
```
Comment (lukebrawleysmith) [inline packages/stats/gbstats/bayesian/tests.py:181]: nit: would be easier to read as 
```1 / scaled_prior_effect.variance if scaled_prior_effect.proper else 0```

**Review by lukebrawleysmith — APPROVED** [review]
2 minor comments

File: packages/stats/gbstats/bayesian/tests.py (hunk 2 above)
Comment (lukesonnet) [reply packages/stats/gbstats/bayesian/tests.py:181]: yes, much better. I was being hasty. thanks

File: packages/stats/gbstats/bayesian/tests.py (hunk 1 above)
Comment (lukesonnet) [reply packages/stats/gbstats/bayesian/tests.py:156]: Sure, we just have other problems with negative metrics but I think it should be fine here.

**Pull request merged**